website/versioned_docs/version-5.x/layout-definition.md

```diff
@@ -162,20 +162,29 @@ The same application may appear multiple times in your layout, under different r
 <application name="appName" props="myProp,authToken"></application>
 ```
 
+#### application config using JSON
```

Comment:
Since there is no header for the application config with HTML, I don't think there should be a header for application config using JSON.

I intentionally didn't add the subheader for these to avoid the sidebar having too many entries

Reply:
Yeah, ignore this PR.